cdip_admin/emails/templates/invite_email.html

```diff
@@ -1,20 +1,61 @@
+<!DOCTYPE html>
 <html lang="en">
 <head>
-  <meta charset="utf-8">
-  <title>Welcome</title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Welcome to Gundi</title>
+    <style>
+        body {
+            font-family: Arial, sans-serif;
+            line-height: 1.6;
+            color: #333;
+        }
+        .container {
+            max-width: 600px;
+            margin: auto;
+            padding: 20px;
+        }
+        h2 {
+            color: #004E26;
+        }
+        .container p {
+            margin-bottom: 10px;
+        }
+        .container ul {
+            padding-left: 20px;
+        }
+        .container a {
+            color: #52B34C;
+            text-decoration: none;
+        }
+        .container a:hover {
+            text-decoration: underline;
+        }
+    </style>
 </head>
 <body>
-    <h2>Welcome {{name}}</h2>
-    <p>
-        You have been invited to collaborate with {{organization}} in Gundi.
-    </p>
-    {% if is_new_user %}
-    <p>
-        You will need to reset your password clicking in "Forgot Password?".
-    </p>
-    {% endif %}
-    <p>
-        <a href="{{action_url}}">Go to Gundi</a>
-    </p>
+    <div class="container">
+        <h2>Welcome {{name}}</h2>
+        <p>
+            You’ve been invited to join {{organization}} in <a href="https://projectgundi.org/">Gundi</a>, the free and open-source integration platform developed by the EarthRanger program in collaboration with WCS and WPS that helps you create and manage connections between physical and virtual sensors with conservation tools.
```

Comment:
Can I suggest this edit.

```suggestion
            You’ve been invited to join {{organization}} in <a href="https://projectgundi.org/">Gundi</a> the free and open-source integration platform for conservation. It's developed by EarthRanger in collaboration with Wildlife Conservation Society and Wildlife Protection Solutions to help you connect your data to your conservation platform and tools.
```

Reply:
I like the idea of two shorter sentences instead of a single long sentence.